### PR TITLE
Doc: How we schedule meetings

### PR DESCRIPTION
**Pull request by patcon**
Re: https://github.com/hyphacoop/organizing/pull/56/files#r297862862

(Half thinking this convo should happen in Loomio as our main decision-making place, rather than this issue. Or maybe just a convo about whether the handbook expects to be explicit or high-level. Both have merits!) cc @dcwalk

### CODE REVIEW

**Comment by patcon** [comment]
Wondering if I might be being too specific. Not done yet, but eager for feedback on that!

File: meetings.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    - [Standup](#standup)
+    - [Full](#full)
+  - [Holds](#holds)
+- [Scheduling](#scheduling)
+- [Hosting](#hosting)
+- [Expectations](#expectations)
+
+## Types
+
+### All Hands
```
Comment (dcwalk) [inline meetings.md]: This structure seems confusing, can we just have "All-hands full meeting" and "All-hands standup" and reduce one layer of heading?

File: meetings.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+workload (inside and outside co-op), and needs of organization-wide
+initiatives.
+
+:calendar: **Schedule:** We currently meet every 2 weeks, alternating between
+format of a short **standup** and a **full** meeting.
+
+#### Standup
+:clock10: **Duration:** 15 minutes
+
+This format involves...
```
Comment (dcwalk) [inline meetings.md]: see proposal above

File: meetings.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+#### Full
+:clock10: **Duration:** 60 minutes
+
+This format involves...
+
+### Holds
```
Comment (dcwalk) [inline meetings.md]: This doesn't really seem like a meeting type, can we just have this as a top level item? "Calendar Holds"

File: meetings.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+Holds are unclaimed spots in our shared schedule that we try to leave
```
Comment (dcwalk) [inline meetings.md]:
```suggestion
Holds are unclaimed spots in our shared schedule when most members are available that we try to leave
```

File: meetings.md (hunk 3 above)
Comment (dcwalk) [reply meetings.md]: Do you want to add something about how holds get decided? 
"periodically re-poll members to find the best time"

File: meetings.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+open. They could be used for:
+
+- Discussions requiring the whole organization
+- Last-minute or urgent meetings, for which a new scheduling is
+  difficult
+
+## Scheduling
```
Comment (dcwalk) [inline meetings.md]: This seems confusing/long to have a heading that will just direct to another area, could this + hosting be made one comment at the top?

"See "Sched" and "Hosting" meeting protocols" for specific instructions

File: meetings.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+See the [_"Scheduling a meeting"_ protocol][scheduling-protocol].
+
+## Hosting
+
+See the [_"Hosting a meeting"_ protocol][hosting-protocol].
+
+   [scheduling-protocol]: /protocols.md#scheduling-a-meeting
+   [hosting-protocol]: /protocols.md#hosting-a-meeting
+
+## Expectations
```
Comment (dcwalk) [inline meetings.md]: We could take this from the december retreat doc, but I think actually we don't have these yet. So maybe this is premature?

File: protocols.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
+# Protocols
+
+> Protocols are the "attitudes" or "the manner in which one approaches each and
```
Comment (dcwalk) [inline protocols.md]: See my comment on "protocol" as a term (feeling a little technologizing / scientizing), that said I'm totally down with gug of the idea of protocols

File: protocols.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+>
+> -- [Lab Book, Civic Laboratory for Environmental Action Research (CLEAR)][clear-lab-book]
+
+## Scheduling a meeting
+
+Note: Consider whether this meeting might be a global interest to other
+  members. Skip below steps at your discretion for low-stakes topics.
+
+- Try to announce your intention to schedule a meeting on a topic, and ask
+  who's interested to participate. Allow a day for people to respond.
```
Comment (dcwalk) [inline protocols.md]:
```suggestion
  who's interested to participate. Allow _at least_ a day for people to respond.
```

I'd say if our internal turnaround time is 48hours 1 day is a short minimum

File: protocols.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+- Consider who might be most interested, and what timing might work for
+  them.
+  - Check the [monthly `availability` calendar for travel/vacation plans][availability].
+  - Check everyone's [weekly availability `schedules`][schedules].
+    for them.
```
Comment (dcwalk) [inline protocols.md]:
```suggestion
```

File: protocols.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+  - Ask in chat whether anyone who wants to attend has any blocks on any
+    of the options.
+- If short-notice or speed of booking is important:
+  - Claim an open HOLD [:information_source:](/meetings.md#holds) slot.
+    - If time permits or topic isn't an all-hands matter, consider
+      leaving the hold available for someone else.
+    - If claiming a hold, edit the calendar immediately, and announce
+      meeting in chat.
+- If your must find a new timeslot:
+  - Create a [new When2Meet poll](https://www.when2meet.com/).
```
Comment (dcwalk) [inline protocols.md]: Maybe say "Create a new poll" (we currently use [When2Meet](#))

File: protocols.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+  - Try to leave ample time for completion (up to 1 week?).
```
Comment (dcwalk) [inline protocols.md]:
```suggestion
  - Leave ample time for completion (ideally 1 week).
```

File: protocols.md [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+  - Check on non-responders in following days, and poke them.
```
Comment (dcwalk) [inline protocols.md]:
```suggestion
  - Check on non-responders in following days, and send (gentle) reminders as needed.
```

File: protocols.md [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+
+## Hosting a meeting
+
+- **Before** the meeting...
+  - create a stub agenda as soon as possible (can be done before scheduling)
+    - Hint: use [an agenda `template`][template].
+  - add the agenda to the [`meetings` index][meetings].
+  - update the [organizational calendar][calendar] as event scheduling
```
Comment (dcwalk) [inline protocols.md]: Unclear what this specifically means, maybe rephrase? 

"as soon as the event is scheduled"

File: protocols.md [hunk 14]
```diff
@@ -0,0 +1,73 @@
...
+- If your must find a new timeslot:
+  - Create a [new When2Meet poll](https://www.when2meet.com/).
+  - Try to leave ample time for completion (up to 1 week?).
+  - Check on non-responders in following days, and poke them.
+
+## Hosting a meeting
+
+- **Before** the meeting...
+  - create a stub agenda as soon as possible (can be done before scheduling)
+    - Hint: use [an agenda `template`][template].
```
Comment (dcwalk) [inline protocols.md]:
```suggestion
    - Hint: use [an agenda `template`][template]
```

File: protocols.md [hunk 15]
```diff
@@ -0,0 +1,73 @@
...
+  - add the agenda to the [`meetings` index][meetings].
```
Comment (dcwalk) [inline protocols.md]:
```suggestion
  - add the agenda to the [`meetings` index][meetings]
```

File: protocols.md [hunk 16]
```diff
@@ -0,0 +1,73 @@
...
+  - update the [organizational calendar][calendar] as event scheduling
+    progresses.
```
Comment (dcwalk) [inline protocols.md]:
```suggestion
    progresses
```

**Comment by dcwalk** [comment]
> Wondering if I might be being too specific. Not done yet, but eager for feedback on that!

Provided perhaps too granular feedback. I'd say it is probably okay level of detail, tho others probably have thoughts. 

High-level thoughts:
- See discussion elsewhere (I think was chat?) about "protocols" as concept and whether we have others we are drawing on.
- I think that it might be introducing too much structured text when there isn't that much to say... I'd favour reducing headlines/sections if they are only going to have a sentence fragment or link into other areas.

File: meetings.md (hunk 6 above)
Comment (patcon) [reply meetings.md]: Good point. Could we have a practice of leaving placeholders like this in skeleton of doc? I like to idea of leaving intended bits in the public face that we can be self-conscious about, to speak to aspirations even if not current state. Feels like it embodies "living document" ethos, and points to intentions in a way that casual readers can see (e.g., not navigating into our internals tracker for the issue mentioning it)

File: protocols.md (hunk 7 above)
Comment (patcon) [reply protocols.md]: Totally agree. What about stealing "guides" from enspiral?

**Comment by dcwalk** [comment]
@patcon -- next steps on this? Can I help get this to review state before our meeting tomorrow?

File: meetings.md (hunk 1 above)
Comment (patcon) [reply meetings.md]: ok, guess I was thinking this was just skeleton and that things common to both would emerge, but I see your point.

Comment (patcon) [reply meetings.md]: Hm. Even now, there's stuff common to both, so I'm not sure agree with the suggestion. I don't any need to conserve heading levels I suppose?

File: protocols.md (hunk 8 above)
Comment (patcon) [reply protocols.md]:
```suggestion
  who's interested to participate. Allow _at least_ 48 hours for people to respond.
```

Your commit suggestion didn't align with my interpretation of your concern. Should I accept mine or yours? :)

Comment (patcon) [reply protocols.md]: (Also, in general, should we air toward hours or days with this stuff -- specificity or intentional generality?)

File: meetings.md (hunk 5 above)
Comment (patcon) [reply meetings.md]: Good pattern.

File: meetings.md (hunk 3 above)
Comment (patcon) [reply meetings.md]: Done!

File: meetings.md (hunk 4 above)
Comment (patcon) [reply meetings.md]: Added in another commit due to conflict

File: protocols.md (hunk 7 above)
Comment (patcon) [reply protocols.md]: used "guides" though we're being a little more specific than enspiral treats guides

File: protocols.md (hunk 10 above)
Comment (patcon) [reply protocols.md]: Good call. Committed locally.

File: protocols.md (hunk 11 above)
Comment (patcon) [reply protocols.md]: Committed

File: protocols.md (hunk 13 above)
Comment (patcon) [reply protocols.md]: Ah, these used to be closer in an earlier version, but the "Scheduling a meeting" section advises holding a few slots (like you did for design jam). The idea was that the calendar should always reflect options and finalized times.

In light of that, maybe "update the organizational calendar as early as possible with candidate and finalized time slots"?

File: protocols.md (hunk 14 above)
Comment (patcon) [reply protocols.md]: done, and related question below :)

**Comment by patcon** [comment]
Quick Q dcwalk: what's your rule of thumb for lists -- re: capitalization and end-of-line punctuation?

**Comment by patcon** [comment]
> See discussion elsewhere (I think was chat?) about "protocols" as concept and whether we have others we are drawing on.

Presumably you mean this: https://matrix.to/#/!wvFLvXHaNrxWWwFOKj:tomesh.net/$1562129800918AzGXq:tomesh.net?via=tomesh.net&via=aseriesoftubez.com&via=matrix.org

> I think protocols as a format for thinking about this stuff works (but I'm not sold on the name/concept... Sometimes it feels technologizing as a term.) I also think we don't need to use it and could just say "handbook".... And cite inspo

Went with "Guides" after your comments, as that seemed perhaps better lingo

**Comment by patcon** [comment]
> I think that it might be introducing too much structured text when there isn't that much to say...

I'm not sure I agree that there isn't much to say. However, I do understand if it's "too much" in terms of too overbearing. But yeah, I don't feel that it's "too much" as in "everyone does this already". I certainly don't do all this yet, but I'm curious whether others aspire to do similar-ish things. I suppose I was hoping we stored our idealized practices/suggestions here? :)

File: roles.md [hunk 17]
```diff
@@ -4,16 +4,7 @@
...
--  chairing all hands meetings. This includes...
-   - preparing a pad in advance so we start on time
-   - asking for agenda items (start of meeting)
-   - facilitating discussion
-   - keeping time
-- adding reminders to calendar after meeting
-- reviewing and merging notes
-- as needed, set up loomio theads and decisions
-- as needed, sending reminders of action items
+**Tasks** are described in Guide for [hosting a meeting](#hosting-a-meeting).
```
Comment (patcon) [inline roles.md]: Is this migration an ok thing? I did it when I realized the cultivator role had some things that perhaps could belong in a general guide

Comment (patcon) [reply roles.md]:
```suggestion
**Tasks** are described in Guide for [hosting a meeting](/guides.md#hosting-a-meeting).
```

File: guides.md [hunk 18]
```diff
@@ -0,0 +1,83 @@
...
+instructions; they are ways that we establish order and maintain practices
+across our group [...] -- They define the way we ought to proceed or behave in
+different situations. As such, they are normative, or premised on values,
+morals, and an idea of how things ought to be done. They are a manifestation of
+our values.
+
+## Scheduling a meeting
+
+Note: Consider whether this meeting might be a global interest to other
+members. Skip below steps at your discretion for low-stakes topics.
```
Comment (dcwalk) [inline guides.md]:
```suggestion
members. Skip the steps below at your discretion for low-stakes topics.
```

File: guides.md [hunk 19]
```diff
@@ -0,0 +1,83 @@
...
+
+- Try to announce your intention to schedule a meeting on a topic, and ask
```
Comment (dcwalk) [inline guides.md]:
```suggestion
- Try to announce your intention to schedule a meeting on a topic and ask
```

File: guides.md [hunk 20]
```diff
@@ -0,0 +1,83 @@
...
+  who's interested to participate. Allow a day for people to respond.
```
Comment (dcwalk) [inline guides.md]:
```suggestion
  who's interested in participating. Allow a day for people to respond.
```

File: guides.md [hunk 21]
```diff
@@ -0,0 +1,83 @@
...
+- Consider who might be most interested, and what timing might work for
```
Comment (dcwalk) [inline guides.md]:
```suggestion
- Consider who might be most interested and what timing might work for
```

File: guides.md [hunk 22]
```diff
@@ -0,0 +1,83 @@
...
+  them.
+  - Check the [monthly `availability` calendar for travel/vacation plans][availability].
+  - Check everyone's [weekly availability `schedules`][schedules].
+- If 2-3 candidate timeslots feel obvious from looking at internal
+  availabilities:
```
Comment (dcwalk) [inline guides.md]: clarify what "internal availabilities" means (I'm really struggling), could we say "people's availability"?

File: guides.md [hunk 23]
```diff
@@ -0,0 +1,83 @@
...
+  - Reserve them all in [Hypha calendar][calendar] immediately, for example: `HOLD:
+    Infrastructure Meeting (Option 1)`
+  - Ask in chat whether anyone who wants to attend has any blocks on any
+    of the options.
+- If short-notice or speed of booking is important:
+  - Claim an open HOLD [:information_source:](/meetings.md#holds) slot.
+    - If time permits or topic isn't an all-hands matter, consider
+      leaving the hold available for someone else.
+    - If claiming a hold, edit the calendar immediately, and announce
```
Comment (dcwalk) [inline guides.md]:
```suggestion
    - If claiming a hold, edit the calendar immediately and announce
```

File: guides.md [hunk 24]
```diff
@@ -0,0 +1,83 @@
...
+      meeting in chat.
+- If your must find a new timeslot:
+  - Create a new poll. (We currently use [When2Meet](https://www.when2meet.com/).)
```
Comment (dcwalk) [inline guides.md]:
```suggestion
  - Create a new poll (We currently use [When2Meet](https://www.when2meet.com/)).
```

File: guides.md [hunk 25]
```diff
@@ -0,0 +1,83 @@
...
+  - Leave ample time for completion (ideally 1 week).
+  - Check on non-responders in following days, and send (gentle)
```
Comment (dcwalk) [inline guides.md]:
```suggestion
  - Check on non-responders in following days and send (gentle)
```

File: guides.md [hunk 26]
```diff
@@ -0,0 +1,83 @@
...
+  - keep time
+  - call attention to action items, upcoming decisions and discussions
+- **After** the meeting...
+  - add important dates/reminders to calendar
+  - review and merge notes
+  - update [meeting index][meetings] as best location changes
+    (hackmd => review (PR) => file)
+  - as needed, set up loomio threads and decisions
+  - as needed, send reminders of action items
+  - future: migrate action items into task tracker
```
Comment (dcwalk) [inline guides.md]: Unclear how "future" is being used here, can we just get rid of "future" given task tracking immanent arrival?

File: meetings.md (hunk 6 above)
Comment (dcwalk) [reply meetings.md]: Excessive placeholders doesn't feel living doc moreso "house still under construction (potentially abandoned)" to me, but I don't feel strongly about it in this file

File: roles.md (hunk 17 above)
Comment (dcwalk) [reply roles.md]: This just seems like more clicking around honestly. I would rather read from my _ROLE_ as opposed to _TASK_ if I'm trying to quickly get up to speed (this comes from my experience working from many a training manual and incomplete job handovers)

File: guides.md [hunk 27]
```diff
@@ -0,0 +1,83 @@
...
+      leaving the hold available for someone else.
+    - If claiming a hold, edit the calendar immediately, and announce
+      meeting in chat.
+- If your must find a new timeslot:
+  - Create a new poll. (We currently use [When2Meet](https://www.when2meet.com/).)
+  - Leave ample time for completion (ideally 1 week).
+  - Check on non-responders in following days, and send (gentle)
+    reminders as needed.
+
+## Hosting a meeting
```
Comment (dcwalk) [inline guides.md]: See my point below about moving this -- would not be my preference

File: protocols.md (hunk 8 above)
Comment (dcwalk) [reply protocols.md]: go for yours. don't think days/hours matters with this given our specificity elsewhere

File: protocols.md (hunk 13 above)
Comment (dcwalk) [reply protocols.md]: :+1:

**Comment by dcwalk** [comment]
> Quick Q dcwalk: what's your rule of thumb for lists -- re: capitalization and end-of-line punctuation?

We have done NO trailing periods and I think only capitalization for first level lists elsewhere.

I am pretty against trailing periods as they are extra in my books. I don't care about capitalization, rather I care about consistency

File: roles.md (hunk 17 above)
Comment (patcon) [reply roles.md]: Thanks for feedback! But.... I know less clicking is nice, but these responsibilities aren't just for the all-hands cultivator, right?

Can you express more clearly the sort of thing you're hoping we could do to accommodate your concern? I'm interpreting it as "let's keep slightly different versions in both places", which is unexpected :)

File: guides.md (hunk 26 above)
Comment (patcon) [reply guides.md]: Good point :)

Comment (patcon) [reply guides.md]:
```suggestion
  - migrate action items into task tracker
```

File: protocols.md (hunk 13 above)
Comment (patcon) [reply protocols.md]:
```suggestion
  - update the [organizational calendar][calendar] as early as possible with candidate and finalized time slots
```

File: guides.md [hunk 28]
```diff
@@ -0,0 +1,83 @@
...
+
+- **Before** the meeting...
+  - create a stub agenda as soon as possible (can be done before scheduling)
+    - Hint: use [an agenda `template`][template]
+  - add the agenda to the [`meetings` index][meetings]
+  - update the [organizational calendar][calendar] as event scheduling
```
Comment (patcon) [inline guides.md]:
```suggestion
  - update the [organizational calendar][calendar] as early as possible with candidate and finalized time slots
```

File: guides.md (hunk 22 above)
Comment (patcon) [reply guides.md]: "internal availabilities" was just reference to one standing internal when2meet for members. i'm just trying to find language to avoid saying "look at when2meet"... which itself can be a "fresh" one, or using that standing internal one. As in, when scheduling with Jutta, I didn't blow open the scheduling to include everyone, but I checked our "internal availability" on that standing when2meet.

Is this worth capturing here? Happy to use whatever lingo, or even to just drop it as TMI :)

Comment (patcon) [reply guides.md]: Maybe these could use clearer lingo...

- "availability calendar" (gcal) vs "availability schedule" (when2meet)?

File: guides.md (hunk 24 above)
Comment (patcon) [reply guides.md]: oh jeez, my intuitions on this stuff are different. I believe punctuation is fine to go inside when the whole sentence is parenthetical, right?
https://english.stackexchange.com/a/11129/55389

I'm less inclined to do it your suggested way, unless it's of the structure "(and ...)" or "(when ... )". But as it stands, it's a full sentence in the parens :)

Thoughts?

File: meetings.md (hunk 6 above)
Comment (patcon) [reply meetings.md]: will remove :+1:

File: guides.md [hunk 29]
```diff
@@ -0,0 +1,83 @@
...
+morals, and an idea of how things ought to be done. They are a manifestation of
+our values.
+
+## Scheduling a meeting
+
+Note: Consider whether this meeting might be a global interest to other
+members. Skip the steps below at your discretion for low-stakes topics.
+
+- Try to announce your intention to schedule a meeting on a topic and ask
+  who's interested in participating. Allow a day for people to respond.
```
Comment (patcon) [inline guides.md]:
```suggestion
  who's interested in participating. Allow _at least_ 48 hours for people to respond.
```

File: guides.md (hunk 24 above)
Comment (dcwalk) [reply guides.md]: If it is a full sentence than it shouldn't be in a parenthetical, as I'm used to them explaining/qualifying and doing so while signalling it is set apart from of the flow of writing. This definition roughly corresponds to my understanding: https://www.vocabulary.com/dictionary/parenthetical

File: roles.md (hunk 17 above)
Comment (dcwalk) [reply roles.md]: I'm unsure I understand why my proposal would require keeping info in two places, can we have the "role" be the page where everything lives?

**Comment by dcwalk** [comment]
Hey! I think we could use like 15mins and basically land this, maybe after the call tonight? :dancer:

**Comment by dcwalk** [comment]
Hey! I think we could use like 15mins and basically land this, maybe after another call @patcon ?

File: roles.md (hunk 17 above)
Comment (patcon) [reply roles.md]: But the cultivator isn't the only role that will ever host a meeting. Someone who's not cultivator might need to schedule a meeting, and doesn't it make more sense to point them to "this is where we doc how we schedule meetings", instead of saying "looking at the things the cultivator does for meetings"

Maybe I'm misunderstanding the scope of what a cultivator is though. Is "cultivator" a term for a role in any meeting, or is it a term for a role in the organization / working group?

File: guides.md [hunk 30]
```diff
@@ -0,0 +1,83 @@
...
+
+Note: Consider whether this meeting might be a global interest to other
+members. Skip the steps below at your discretion for low-stakes topics.
+
+- Try to announce your intention to schedule a meeting on a topic and ask
+  who's interested in participating. Allow _at least_ 48 hours for people to respond.
+- Consider who might be most interested and what timing might work for
+  them.
+  - Check the [monthly `availability` calendar for travel/vacation plans][availability].
+  - Check everyone's [weekly availability `schedules`][schedules].
```
Comment (patcon) [inline guides.md]:
```suggestion
  - Check everyone's [availability `schedules`][schedules].
```

File: guides.md [hunk 31]
```diff
@@ -0,0 +1,83 @@
...
+## Scheduling a meeting
+
+Note: Consider whether this meeting might be a global interest to other
+members. Skip the steps below at your discretion for low-stakes topics.
+
+- Try to announce your intention to schedule a meeting on a topic and ask
+  who's interested in participating. Allow _at least_ 48 hours for people to respond.
+- Consider who might be most interested and what timing might work for
+  them.
+  - Check the [monthly `availability` calendar for travel/vacation plans][availability].
```
Comment (patcon) [inline guides.md]:
```suggestion
  - Check the [`availability` calendar for travel/vacation plans][availability].
```

File: guides.md [hunk 32]
```diff
@@ -0,0 +1,83 @@
...
+  - Check everyone's [weekly availability `schedules`][schedules].
+- If 2-3 candidate timeslots feel obvious from looking at internal
```
Comment (patcon) [inline guides.md]:
```suggestion
- If 2-3 candidate timeslots feel obvious from looking at people's
```

File: guides.md (hunk 22 above)
Comment (patcon) [reply guides.md]: dc and i spoke and sorted 🌮 #goodenoughtada

**Comment by patcon** [comment]
@dcwalk ready for final review. thanks a billion!
